### DISCUSSION TITLE
Package distribution channels

### DISCUSSION THREAD

**Discussion by nbolton** [deskflow/deskflow#7603] [General]
I'm currently working out the best strategy for Synergy and Deskflow package distribution through channels like `brew` and `choco`. 

I think perhaps a new package for Deskflow makes sense rather than trying to get the packages renamed.

PRs:
- https://github.com/Homebrew/homebrew-core/pull/195267
- https://github.com/teknowledgist/Chocolatey-packages/issues/108

**Comment by adrianinsaval** [comment]
are you planning on publishing deskflow to flathub?

**Reply by nbolton** [reply]
Yes, that is our intention: #7593

**Comment by sithlord48** [comment]
Created our own [homebrew-tap ](https://github.com/deskflow/homebrew-tap) with casks
